internal/server/cluster/membership.go

```diff
@@ -593,6 +593,8 @@ func NotifyHeartbeat(state *state.State, gateway *Gateway) {
 		// Wait for heartbeat to finish and then release.
 		// Ignore staticcheck "SA2001: empty critical section" because we want to wait for the lock.
 		gateway.HeartbeatLock.Lock()
+		//nolint:all
+		//lint:ignore SA2001 we want to wait for the lock
```

Comment:
The comment above (`// Ignore staticcheck ...`) can be removed given this one just repeats it